app/service/auth_svc.py

```diff
@@ -123,14 +128,35 @@ async def _check_credentials(user_map, username, password):
     async def _ldap_login(self, username, password):
         server = ldap3.Server(self.ldap_config.get('server'))
         dn = self.ldap_config.get('dn')
-        userattr = self.ldap_config.get('userattr') or 'uid'
-        userstring = '{}={},{}'.format(userattr, username, dn)
-        with ldap3.Connection(server, user=userstring, password=password) as conn:
+        user_attr = self.ldap_config.get('user_attr') or 'uid'
+        user_string = '{}={},{}'.format(user_attr, username, dn)
```

Comment:
nothing wrong here, but for project consistency use the % string formatting.

Reply:
Done